netjson-monitoring.lua

```diff
@@ -119,6 +119,27 @@ load_average = {tonumber(loadavg_output[1]),
                 tonumber(loadavg_output[2]),
                 tonumber(loadavg_output[3])}
 
+function parse_disk_usage()
+    file = io.popen('df')
+    disk_usage_info = {}
+    for line in file:lines() do
+        if line:sub(1, 10) ~= 'Filesystem' then
+            filesystem, memory, used, available, percent, location = line:match("(%S+)%s+(%S+)%s+(%S+)%s+(%S+)%s+(%S+)%s+(%S+)")
+            table.insert(disk_usage_info, {
+                name = filesystem,
+                -- in 1k blocks
+                size = memory,
+                used_space = used,
```

Comment:
It would be nice to send these data as numbers. You can use tonumber(used). I have quickly tried it but it failed so you will need to investigate why :wink: